README.md

```diff
@@ -1,6 +1,59 @@
-# curriculum-cr-training-mobile-menu
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
+# curriculum-cr-training-mobile-menu
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+> This repo host an initial code for the tasks performed during the Code Reviewers training.
 
 Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+
+
+
+## Built With
+
+- HTML
+- CSS
+- Technologies used
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- an ide such as vscode, sublime
+### Setup
+git clone this repo
+~ git clone : [ssh](git@github.com:Nyame-Wolf/CR_mobile_menu.git)
+            : [http](https://github.com/Nyame-Wolf/CR_mobile_menu.git)
+### Usage
+~ cd this repo
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@Nyame-Wolf](https://github.com/Nyame-Wolf)
+- Twitter: [@Mumenyam](https://twitter.com/Mumenyam)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mumenya-nyamu-software-engineer/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Nyame-Wolf/CR_mobile_menu/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- [microverse cr_repo](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md)
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [OPTIONAL] It's a good idea to erase this line of code every time that you customize the README template. :sunglasses: